examples/Papers/cat_qubits/readme.md

```diff
@@ -0,0 +1,71 @@
+- [5] Building a fault-tolerant quantum computer using concatenated cat codes :https://arxiv.org/pdf/2012.04108.pdf
+- [6] Dynamically protected cat-qubits:
+a new paradigm for universal quantum computation : https://iopscience.iop.org/article/10.1088/1367-2630/16/4/045014
+
+
+# Introduction
+
+NISQ devices rely on qubits that are still subject to all sorts of noisy processes, destroying quantum information and therefore interfering with the coherent computations delivered by the promises of having fault tolerance. Among the multiple platforms available for quantum computing today, one stands as particularly interesting due to its capacity to perform partial "autonomous" quantum error correction, meaning that the natural unitary evolution and associated engineered dissipation processes
+are enough to preserve the information encoded in the system from Markovian errors.
+This paradigm relies on the control of the so-called classical states of light, or "coherent" states of light. 
+The goal of this overview is to provide the basics of this encoding, and how QUA can be used to perform calibration and control of the associated hardware, presented in reference [3] above, which will be the main paper of reference for the QUA scripts.
+
+# Reminder on coherent states of lights
+
+The cat qubit is defined upon specific states of lights characterizing the modes in a high-Q cavity that behaves like a quantum harmonic oscillator (QHO). Provided the eigenbasis $$\{|n\rangle, n\in \mathcal{N}\}$$ of the Hamiltonian characterizing the QHO, we define the coherent state of complex amplitude $$\alpha$$ as :
+$$|\alpha\rangle:=e^{-|\alpha|^2/2} \displaystyle \sum_n\frac{\alpha^n}{\sqrt{n!}}|n\rangle$$
+
+The property of those states are that they constitute eigenstates of the annihilation operator  $$\hat{a}$$ of the QHO, that is $$\hat{a}|\alpha\rangle=\alpha|\alpha\rangle$$.
+One can easily show that the average photon number $$\langle\hat{a}^{\dagger}\hat{a}\rangle\equiv\langle\alpha|\hat{a}^{\dagger}\hat{a}|\alpha\rangle$$ on this state is given by $$|\alpha|^2$$.
+Moreover, the dot product between two coherent states $$|\alpha\rangle$$ and $$|\beta\rangle$$ decreases exponentially with the norm of the respective photon numbers : $$\langle\alpha|\beta\rangle=e^{-\frac{1}{2}(|\alpha|^2+|\beta|^2-2\alpha^*\beta)}$$
+This means that the states $$|\alpha\rangle$$ and $$|-\alpha\rangle$$ are quasi-orthogonal if $$|\alpha|^2\gg 1$$: $$\langle\alpha|-\alpha\rangle=e^{-2|\alpha|^2}$$.
+Finally, a coherent state can be obtained from the vacuum state of the QHO by applying the displacement operation:
+$$\hat{D}(\alpha)|0\rangle=e^{\alpha\hat{a}^\dagger-\alpha^*\hat{a}}|0\rangle=|\alpha\rangle$$.
+
+
+# Encoding the qubit
+![Bloch sphere of the cat qubit [5]](/cat_qubit_sphere.png)

```

Comment:
This image could not be found. see logs: https://app.netlify.com/sites/qua-libs/deploys/60abad3d6871a1000704e2ae